x/auth/vesting/types/vesting_account_test.go

```diff
@@ -578,6 +563,146 @@ func TestTrackUndelegationPeriodicVestingAcc(t *testing.T) {
 	require.Equal(t, sdk.Coins{sdk.NewInt64Coin(stakeDenom, 25)}, pva.DelegatedVesting)
 }
 
+func TestGetVestedCoinsPermLockedVestingAcc(t *testing.T) {
+	now := tmtime.Now()
+	endTime := now.Add(1000 * 24 * time.Hour)
+
+	_, _, addr := testdata.KeyTestPubAddr()
+	origCoins := sdk.Coins{sdk.NewInt64Coin(feeDenom, 1000), sdk.NewInt64Coin(stakeDenom, 100)}
+	bacc := authtypes.NewBaseAccountWithAddress(addr)
+
+	// require no coins are vested until schedule maturation
```

Comment:
comment makes no sense
```suggestion
	// require no coins are vested
```